V1/DTOs/Session/CreateSessionRequest.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace CoviIDApiCore.V1.DTOs.Session
+{
+    public class CreateSessionRequest
+    {
+        public string MobileNumber { get; set; }
+        public string MobileNumberReference { get; set; }
```

Comment:
The reference will not be available here. So we're going to have to remove the 27 in the front and add a 0.